README.md

```diff
@@ -3,7 +3,7 @@
 > A generic backend with a client API for Offline First applications
 
 [![Build Status](https://travis-ci.org/hoodiehq/hoodie.svg?branch=master)](https://travis-ci.org/hoodiehq/hoodie)
-[![Coverage Status](https://coveralls.io/repos/hoodiehq/hoodie/badge.svg?branch=master)](https://coveralls.io/r/hoodiehq/hoodie?branch=master)
+[![Coverage Status](https://coveralls.io/github/hoodiehq/hoodie?branch=master)](https://coveralls.io/r/hoodiehq/hoodie?branch=master)
```

Comment:
you fixed the wrong URL, sorry :) `https://coveralls.io/repos/hoodiehq/hoodie/badge.svg?branch=master` is the URL of the image, which is correct. What we want to change is the URL of link, meaning we want to change `https://coveralls.io/r/hoodiehq/hoodie?branch=master` to `https://coveralls.io/github/hoodiehq/hoodie`